src/utils/api/Api.js

```diff
@@ -138,14 +138,14 @@ class Api {
     const { data } = store.getState().keycloak;
 
     const publicLayoutUrl = (
-      data.ENV_LAYOUT_PUBLICURL ||
-      (  process.env.ENV_LAYOUT_PUBLICURL ) ||
+      ( process.env.ENV_LAYOUT_PUBLICURL ) ||
+      ( data.ENV_LAYOUT_PUBLICURL ) ||
```

Comment:
We should update this to be `data && data.[...]` to ensure the app never crashes if the data / config is falsy. Apart from that, everything looks good!